collect_app/src/main/res/values/strings.xml

```diff
@@ -13,6 +13,7 @@
 <resources xmlns:tools="http://schemas.android.com/tools" tools:locale="en"
            tools:ignore="MissingTranslation">
 <string name="accept_location">Record Location</string>
+    <string name="show">show password</string>
```

Comment:
There shouldn't be any leading space. The resource name should be show_password. It should be added at the end. The first letter should be capitalized: "Show password".